spec/features/web_helper.rb

```diff
@@ -1,3 +1,5 @@
+require 'date'
```

Comment:
is that necessary? All files should have been required by the spec helper when it call on app/app.rb